devenv/lib/proc.py

```diff
@@ -88,25 +56,27 @@ def run(
     if constants.DEBUG:
         xtrace(cmd)
     try:
-        proc = subprocess_run(cmd, check=True, stdout=_stdout, cwd=cwd, env=env)
-        if _stdout:
+        proc = subprocess_run(
+            cmd, check=True, stdout=PIPE, stderr=PIPE, cwd=cwd, env=env
```

Comment:
are there no places where we run subprocesses and let the output go directly to the user? (make install-py-dev maybe?)

Reply:
yeah you're right - i'm gonna rollback the "always capture stdout" part it's not a good idea